metrics/custom.go

```diff
@@ -4,49 +4,59 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
+const (
+	promNS = "gopherciser"
+)
+
 // GopherActions action counter
 var GopherActions = prometheus.NewCounterVec(
 	prometheus.CounterOpts{
-		Name: "gopherciser_actions_total",
-		Help: "Number of gopherciser actions and their result. DOES NOT USE LABEL!",
+		Namespace: promNS,
```

Comment:
Namespace will be hardcoded to "promNS" independent of where or how we run this.
How will it be used?

Reply:
```golang
promNS = "gopherciser"
```
so gopherciser, but yes don't know how namespace effects either

Reply:
@HvonP it's already hardcoded to `gopherciser` effectively...

`Namespace` is used as a common prefix to the final name. In this particular case, since Gopherciser is a specific application, it doesn't make sense to ever use a different one. But it reduces duplication.

There's also a `Subsystem` key which can be used sometimes, but isn't really necessary in this particular case.

See the docs on those here: https://pkg.go.dev/github.com/prometheus/client_golang/prometheus?tab=doc#Opts